e2e/common/utilities/test-util.ts

```diff
@@ -7,76 +7,82 @@ declare const allure: any;
 
```

Comment:
Copy also the js-console.ts from the HemoHub/Modulab source code.

Reply:
It's already in